client/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactElement } from "react";
+import { RouteComponentProps } from "react-router";
+import Avatar from "@material-ui/core/Avatar";
+import Paper from "@material-ui/core/Paper";
+import useApi from "../../hooks/useApi";
+import Loading from "../Routes/Loading";
+import Error from "../Error";
+import ShowComments from "./ShowComments";
+import useStyles from "./Profile.styles";
+
+interface UrlParam {
+  username: string;
+}
+interface Comment {
+  movieName: string;
+  date: number;
+  name: string;
+  body: string;
+  stars: number;
+  _id: number;
+}
+
+const Profile = ({
+  match: {
+    params: { username }
+  }
+}: RouteComponentProps<UrlParam>): ReactElement => {
+  const { resData: data, loading, error } = useApi(`/user/${username}`, {
+    hotReload: true
+  });
+  const classes = useStyles({});
+  // const timestampToString = (date: string) => {
+  //   const date = new Date(date);
+  //   const hours = date.getHours();
+  // };
+  if (loading) {
+    return <Loading />;
+  }
+  if (error) {
+    return <Error />;
+  }
+  return (
```

Comment:
Add one empty line between two block of code for more clarity
```suggestion

    if (loading) {
    return <Loading />;
  }
  
  if (error) {
    return <Error />;
  }
  
  return (
```